src/Core/Termination.idr

```diff
@@ -410,7 +410,7 @@ export
 calculateSizeChange : {auto c : Ref Ctxt Defs} ->
                       FC -> Name -> Core (List SCCall)
 calculateSizeChange loc n
-    = do log "termination.sizechange" 5 $ "Calculating Size Change: " ++ show n
+    = do log "totality.sizechange" 5 $ "Calculating Size Change: " ++ show n
```

Comment:
This should probably be `totality.termination.sizechange` for uniformity
or is it an unrelated pass (in which case a different name would be clearer)?

Reply:
I think a different name is required, same for the one for guarded. They don't appear as subcalls to existing functions in the module and are both exported.

Reply:
Tbh I tend to think of these in terms of topics, not call graphs.

Reply:
I've made the appropriate changes, and think that his is good to go.